recipe/build.sh

```diff
@@ -2,29 +2,10 @@
 
 set -x
 
-pushd pkg/CaratInterface
+pushd pkg/caratinterface
     tar pzxf carat.tgz
 popd
 
-# Get an updated config.sub, config.guess and libtool
-for f in $(find $SRC_DIR -name config.sub); do
-    cp $BUILD_PREFIX/share/gnuconfig/config.sub $f
-done
-for f in $(find $SRC_DIR -name config.guess); do
-    cp $BUILD_PREFIX/share/gnuconfig/config.guess $f
-done
-for f in $(find $SRC_DIR -name libtool); do
-    cp $BUILD_PREFIX/bin/libtool $f
-done
-for f in $(find $SRC_DIR -name libtool.m4); do
-    cp $BUILD_PREFIX/share/aclocal/libtool.m4 $f
-    pushd $(dirname $(dirname $f))
-        autoreconf -vfi || true
-    popd
-done
-
-autoreconf -vfi
-
 export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
 export CFLAGS="-g -O3 -fPIC $CFLAGS"
 rm -f $BUILD_PREFIX/bin/curl-config
```

Comment:
In the lines below this you delete the PDFs from the tarball, which causes `make install` to file. Possible solutions:
1. just don't do this
2. do it but only *after* installing (i.e. delete the installed files)
3. patch the `install-doc` target to not install them
4. patch the `install` target to not trigger `install-doc`

Reply:
You also delete the `.lab` files which means that if someone were to try to rebuild the documentation of certain packages (those not using GAPDoc for their manuals), then references to the GAP manual would be broken.